go.mod

```diff
@@ -5,55 +5,66 @@ go 1.18
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/Microsoft/go-winio v0.5.2
-	github.com/cenkalti/backoff/v4 v4.1.1
+	github.com/cenkalti/backoff/v4 v4.1.3
 	github.com/containerd/cgroups v1.0.3
-	github.com/containerd/console v1.0.2
-	github.com/containerd/containerd v1.5.13
+	github.com/containerd/console v1.0.3
+	github.com/containerd/containerd v1.6.6
 	github.com/containerd/go-runc v1.0.0
 	github.com/containerd/ttrpc v1.1.0
 	github.com/containerd/typeurl v1.0.2
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/mock v1.6.0
-	github.com/google/go-cmp v0.5.6
+	github.com/google/go-cmp v0.5.7
 	github.com/google/go-containerregistry v0.5.1
 	github.com/google/uuid v1.3.0
 	github.com/linuxkit/virtsock v0.0.0-20201010232012-f8cee7dfc7a3
-	github.com/mattn/go-shellwords v1.0.6
-	github.com/opencontainers/runc v1.0.3
+	github.com/mattn/go-shellwords v1.0.12
+	github.com/open-policy-agent/opa v0.42.2
+	github.com/opencontainers/runc v1.1.2
 	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.8.1
 	github.com/urfave/cli v1.22.2
-	github.com/vishvananda/netlink v1.1.1-0.20201029203352-d40f9887b852
-	github.com/vishvananda/netns v0.0.0-20200728191858-db3c7e526aae
+	github.com/vishvananda/netlink v1.1.1-0.20210330154013-f5de75959ad5
+	github.com/vishvananda/netns v0.0.0-20210104183010-2eb08e3e575f
 	go.etcd.io/bbolt v1.3.6
-	go.opencensus.io v0.22.3
-	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
+	go.opencensus.io v0.23.0
+	golang.org/x/sync v0.0.0-20220601150217-0de741cfad7f
 	golang.org/x/sys v0.0.0-20220412211240-33da011f77ad
-	google.golang.org/grpc v1.40.0
+	google.golang.org/grpc v1.47.0
```

Comment:
All of these were bumped just for the new open-policy-agent dependency?

Reply:
Its the new dependency. Given we merely run "go mod" to update this, I can't comment on what specifically causes a given line to update.

Reply:
Yea it looks like that dep has quite the dependency tree.. it's definitely the one bumping containerd and grpc, and the others are likely a cascade from the new containerd version

Reply:
Did you also run `go mod tidy`?  I've found on past projects that there is a reduced likelihood of merge conflicts between different branches and PRs if `go mod tidy` is never run since the old versions are always captured and can be compared against rather than removed.

Reply:
Ignore my comment, got myself turned around.

Reply:
Yea we have a CI job that should validate that the vendor dir/go.mod is matching after a go mod tidy/vendor flow. Just crazy to see so many bumps from a single dependency